src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.h

```diff
@@ -27,7 +26,13 @@ typedef enum {
     /* 0x04 */ ITEM_ETCETERA_SCALE_SILVER,
     /* 0x05 */ ITEM_ETCETERA_SCALE_GOLD,
     /* 0x06 */ ITEM_ETCETERA_KEY_SMALL,
-    /* 0x07 */ ITEM_ETCETERA_ARROW_FIRE
+    /* 0x07 */ ITEM_ETCETERA_ARROW_FIRE,
+    /* 0x08 */ ITEM_ETCETERA_RUPEE_GREEN_CHEST_GAME,
+    /* 0x09 */ ITEM_ETCETERA_RUPEE_BLUE_CHEST_GAME,
+    /* 0x0A */ ITEM_ETCETERA_RUPEE_RED_CHEST_GAME,
+    /* 0x0B */ ITEM_ETCETERA_RUPEE_PURPLE_CHEST_GAME,
+    /* 0x0C */ ITEM_ETCETERA_HEART_PIECE_CHEST_GAME,
+    /* 0x0D */ ITEM_ETCETERA_SMALL_KEY_CHEST_GAME
```

Comment:
For consistency with `ITEM_ETCETERA_KEY_SMALL`:
```suggestion
    /* 0x0D */ ITEM_ETCETERA_KEY_SMALL_CHEST_GAME
```